packages/rocketchat-lib/server/models/Subscriptions.coffee

```diff
@@ -312,7 +312,6 @@ class ModelSubscriptions extends RocketChat.models._Base
 		update =
 			$set:
 				alert: true
-				open: true
```

Comment:
This was not a bug :) it was intended. If you want a different behaviour, we need to make it configurable.